### PR TITLE
Automation code for the testcase "Add Job Title"

### PR DESCRIPTION
**Pull request by Neethujn**
testcase for add job

### CODE REVIEW

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/LoginTests.java [hunk 1]
```diff
@@ -20,4 +20,6 @@ public void loginTestWithInValidCredentials(){
     }
 
 
+
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/tests/LoginTests.java]: This needs to be changed as it was

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+public class JobTitleTest extends TestBase {
+
+    @Test
+    public void addNewJob( ) {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("admin", "admin123", true, null);
+        HeaderPage headerPage = new HeaderPage(driver);
+        headerPage.selectMenu(MenuOptions.JOB_TITLES);
+
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java]: remove blank lines

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+        JobTitlePage jobPage = new JobTitlePage(driver);
+        jobPage.saveNewJob("Quality Engineer");
+       // String uuid = UUID.randomUUID().toString();
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java]: String uuid = UUID.randomUUID().toString();
 String jobname= "JobTitle Test " + uuid;
 jobPage.saveNewJob(jobname);

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    @Test
+    public void addNewJob( ) {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("admin", "admin123", true, null);
+        HeaderPage headerPage = new HeaderPage(driver);
+        headerPage.selectMenu(MenuOptions.JOB_TITLES);
+
+
+        JobTitlePage jobPage = new JobTitlePage(driver);
+        jobPage.saveNewJob("Quality Engineer");
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java]: rename method from saveNewJob to saveJobAndVerifyDisplayed

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+
+import java.util.List;
+
+public class JobTitlePage extends PageBase {
+
+    WebDriver driver;
+    private final String idJobAddBtn = ".oxd-button--secondary";
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java]: remove text id from variable name as it is confusing
remove blank lines

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    private final String idJobTxtLevel="div[class='oxd-form-row'] input[class*='oxd-input']";
+    private final String idJobSaveBtn="//button[@type='submit']";
+
+    private final String lblAlreadyExistsMessage = "//div[@class='oxd-form-row'] /div";
+    private final String getIdJobCancelBtn= "//div[@class='oxd-form-actions'] /button[1]";
+    private final String tblJob = ".oxd-table-body";
+    //private final String educations = "//div[@class='oxd-table-body'] /div[@class='oxd-table-card']";
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java]: remove comments

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/EducationPage.java [hunk 7]
```diff
@@ -20,6 +20,9 @@ public class EducationPage extends PageBase {
     private final String getIdEduCancelBtn= "//div[@class='oxd-form-actions'] /button[1]";
     private final String tblEducation = ".oxd-table-body";
     private final String educations = "//div[@class='oxd-table-body'] /div[@class='oxd-table-card']";
+
+    // //div[contains(@class, 'oxd-table-body')]/div[contains(@class,'oxd-table-card')]
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/pages/EducationPage.java]: you need to revert

File: pom.xml [hunk 8]
```diff
@@ -55,7 +55,7 @@
                 <configuration>
                     <suiteXmlFiles>
                         <suiteXmlFile>src/test/java/ca/qaguru/orangehrmbatch26/tests/testng.xml</suiteXmlFile>
-                     </suiteXmlFiles>
```
Comment (RahulNair92) [inline pom.xml:58]: revert this back

**Review by RahulNair92 — CHANGES_REQUESTED** [review]

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/LoginTests.java (hunk 1 above)
Comment (Neethujn) [reply src/test/java/ca/qaguru/oranghrmbatch27/tests/LoginTests.java]: done with the changes

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java (hunk 2 above)
Comment (Neethujn) [reply src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java]: removed the blank lines

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java (hunk 3 above)
Comment (Neethujn) [reply src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java]: done

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java (hunk 6 above)
Comment (Neethujn) [reply src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java]: removed

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/EducationPage.java (hunk 7 above)
Comment (Neethujn) [reply src/test/java/ca/qaguru/oranghrmbatch27/pages/EducationPage.java]: reverted the changes

**Review by RahulNair92 — APPROVED** [review]

**Pull request merged**